frontend/pulse3d/src/pages/uploads.js

```diff
@@ -139,7 +127,7 @@ const modalObjs = {
 
```

Comment:
and I just want to double check that all these modals are still working as expected? ensuring no one performs actions on pending jobs and that you can't download or perform interactive analysis on failed jobs

Reply:
I just tested these features and they seem to work.